crates/sui-core/src/authority/authority_store.rs

```diff
@@ -1177,7 +1342,7 @@ impl AuthorityStore {
     ) -> SuiResult {
         trace!(?objects, "delete_locks");
         write_batch.delete_batch(
-            &self.perpetual_tables.owned_object_transaction_locks,
```

Comment:
The function name should also be changed to something like delete live object markers